contracts/token/ERC20/ERC20.sol

```diff
@@ -1,35 +1,204 @@
 pragma solidity ^0.4.24;
 
+import "./IERC20.sol";
+import "../../math/SafeMath.sol";
+
 
 /**
- * @title ERC20 interface
- * @dev see https://github.com/ethereum/EIPs/issues/20
+ * @title Standard ERC20 token
+ *
+ * @dev Implementation of the basic standard token.
+ * https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md
+ * Based on code by FirstBlood: https://github.com/Firstbloodio/token/blob/master/smart_contract/FirstBloodToken.sol
```

Comment:
Let's change this to "originally based" at some point later because it implies that it's still similar to that token when it's been improved/changed multiple times over